packages/shared/src/components/Feed.tsx

```diff
@@ -316,6 +316,18 @@ export default function Feed<T>({
     );
   };
 
+  const onCardBookmark = (post: Post, row: number, column: number) =>
+    onBookmark({
+      post,
+      origin,
+      opts: {
+        row,
+        column,
+        columns: virtualizedNumCards,
+        ...feedAnalyticsExtra(feedName, ranking),
+      },
+    });
```

Comment:
We already have `toggleBookmark` right?
This is preferred to use in this case.

Reply:
Yes, that is used here, was just renamed as `onBookmark`.

Reply:
Why did you rename it, the actual action is toggling the bookmark that seemed more explicit no?

Reply:
I did not renamed it. Has been defined as an alias of that here..